Source/Core/DolphinWX/ISOProperties.cpp

```diff
@@ -1246,8 +1247,8 @@ void CISOProperties::OnComputeMD5Sum(wxCommandEvent& WXUNUSED (event))
 	u64 read_offset = 0;
 	md5_context ctx;
 
-	File::IOFile file(OpenGameListItem.GetFileName(), "rb");
-	u64 game_size = file.GetSize();
+	std::unique_ptr<DiscIO::IBlobReader> file(DiscIO::CreateBlobReader(OpenGameListItem.GetFileName()));
+	u64 game_size = file->GetDataSize();
```

Comment:
Could we rename this to something like GetDiskSize, to better match what it actually does?